docs/user-event/api-pointer.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: pointer
+title: Pointer
+---
+
+```ts
+pointer(input: PointerInput): Promise<void>
+```
+
+The `pointer` API allows to simulate interactions with pointer devices. It
+accepts a single pointer action or an array of them.
+
+```ts
+type PointerInput = PointerActionInput | Array<PointerActionInput>
+```
+
+> Our primary target audience tests per `jest` in a `jsdom` environment and there
+is no layout in `jsdom`. This means that different from your browser the
+elements don't exist in a specific position, layer and size.  
+We don't try to determine if the pointer action you describe is possible at that
+position in your layout.
```

Comment:
No this literally means that there is no specific position, layer, or size. Elements can have IDL properties and/or CSS styles that usually correspond to elements being at a specific position, having specific dimensions, or being above/under each other. But `jsdom` has no layout. The properties and styles exist, but they have no effect.

```js
document.body.innerHTML=`<img style="position: relative; left: 50px;"/>`
$("img").style.left // '50px'
$("img").offsetLeft // 0
```

Reply:
👍 Got it, the comment made it click to me - would it be worth it to also mention it in the docs?

> every character has effective width zero and is at effective offset 0,0. (While there might be properties applied that suggest otherwise but have no effect.)

Reply:
I don't know. I feel like it might be better to write a blog post about this topic explaining the concept and then adding a link. Really explaining this, and what difference it makes to testing might be worth it, but is out of scope for these docs.